mod/cal.php

```diff
@@ -112,11 +111,11 @@ function cal_content(App $a)
 	$owner_uid = intval($owner['uid']);
 	$nick = $owner['nickname'];
 
-	$contact_id = Session::getRemoteContactID($owner['uid']);
+	$contact_id = DI::userSession()->getRemoteContactID($owner['uid']);
```

Comment:
For consistency the method should be renamed to `getRemoteContactId`.
```suggestion
	$contact_id = DI::userSession()->getRemoteContactId($owner['uid']);
```

Reply:
Can I rename this method after all PR at once?
because otherwise I think I will loose my overview :)